src/pages/workspace/WorkspaceMembersPage.js

```diff
@@ -89,6 +92,15 @@ class WorkspaceMembersPage extends React.Component {
         this.getWorkspaceMembers();
     }
 
+    /**
+     * On Search change state of searchValue
```

Comment:
```suggestion
```

This doc comment is not really necessary and should be removed.

Reply:
@marcaaron you mention in previous suggestion This needs a JSDoc since it has an argument.

Reply:
Correct. It needs a doc but not a doc with a comment. Thanks!